modules/ossm-installation-activities.adoc

```diff
@@ -8,9 +8,9 @@
 [id="ossm-installation-activities_{context}"]
 = Operator overview
 
-{SMProductName} requires the following four Operators:
+{SMProductName} requires the following Operators:
 
-* *OpenShift Elasticsearch* - (Optional) Provides database storage for tracing and logging with the {JaegerShortName}. It is based on the open source link:https://www.elastic.co/[Elasticsearch] project.
 * *{JaegerName}* - Provides distributed tracing to monitor and troubleshoot transactions in complex distributed systems. It is based on the open source link:https://www.jaegertracing.io/[Jaeger] project.
 * *Kiali Operator provided by Red Hat* - Provides observability for your service mesh. You can view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
 * *{SMProductName}* - Allows you to connect, secure, control, and observe the microservices that comprise your applications. The {SMProductShortName} Operator defines and monitors the `ServiceMeshControlPlane` resources that manage the deployment, updating, and deletion of the {SMProductShortName} components. It is based on the open source link:https://istio.io/[Istio] project.
+* *OpenShift Elasticsearch* - (Optional) Provides database storage for tracing and logging with the {JaegerShortName}. It is based on the open source link:https://www.elastic.co/[Elasticsearch] project.
```

Comment:
Though OpenShift Elasticsearch is optional, if it is going to be used, it must be installed first. In order to be consistent throughout Service Mesh, OpenShift Elasticsearch must remain first in the list.

Reply:
Revoked this change.